glide.lock

```diff
@@ -0,0 +1,59 @@
+hash: f3d5a216d8cc6c3fa10575636c82f814075fdcf4deb1a83ee02114ba0e25f229
```

Comment:
@kindermoumoute I added gopsutil version to `glide.yaml` and removed `glide.lock` file